### PR TITLE
Stopwatch

### PR DESCRIPTION
**Pull request by stunnerboyone**
- [DEMO LINK](https://stunnerboyone.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
```
Comment (denys-danyliuk) [inline src/index.html:12]: Знімок екрана 2023-08-07 о 12 13 41

File: src/styles/main.scss [hunk 2]
```diff
@@ -1 +1,63 @@
...
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch {
```
Comment (denys-danyliuk) [inline src/styles/main.scss:21]: create separate file and move this block to it, 6'th point in checklist

File: src/styles/main.scss [hunk 3]
```diff
@@ -1 +1,63 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
```
Comment (denys-danyliuk) [inline src/styles/main.scss:25]: 8'th point in checklist

File: src/styles/main.scss [hunk 4]
```diff
@@ -1 +1,12 @@
...
+
+@import "../styles/blocks/stopwatch";
+
+.body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vmin;
+  margin: 0;
+}
```
Comment (xiimiik) [inline src/styles/main.scss:12]: there is no class `.body` in html

**Review by xiimiik — APPROVED** [review]
Nice work!

**Pull request closed without merging**